app/js/filters.js

```diff
@@ -16,7 +17,7 @@ angular.module('myApp.filters', ['myApp.i18n'])
       if (!user || !user.first_name && !user.last_name) {
         return _('user_name_deleted')
       }
-      return user.first_name + (user.last_name ? ' ' + user.last_name : '')
+      return user.first_name ? (user.first_name + (user.last_name ? ' ' + user.last_name : '')) : user.last_name
```

Comment:
It's possible to have no first name if the user is my contact and I have just last name in phonebook for him. So, @bartist's current version is better.